### PR TITLE
(#272, #277) comment and note image reordering

### LINKED ISSUE #272: [Bug] Note 댓글 순서 확인 필요
**내용**

https://diivers.world/api/notes/27/comments/
로 내려오는 결과가 아래 이미지를 참고해보면 created_at 역순으로 오고 있지 않습니다
2번째 3번째 댓글의 created_at을 확인해보면 됩니다~

**버그 재현 절차**

1.
2.
3.

**정상 결과**

옛날꺼부터 최신까지 순서대로 와야합니다!

**이미지**
Screenshot 2024-06-22 at 4 51 33 PM

Screenshot 2024-06-22 at 4 50 47 PM

**테스트 환경**
데스크탑 / 모바일 / 브라우저 등

### LINKED ISSUE #277: [Bug] Note 작성시 올린 이미지 순서와 api에서 내려오는 이미지순서가 반대임
**내용**
Note 작성시에 올린 이미지 순서와 Note를 작성한 후에 api로 불러오는 정보에 포함된 이미지 순서가 반대로 옴

**버그 재현 절차**

1.
2.
3.

**정상 결과**

**이미지**

**테스트 환경**
데스크탑 / 모바일 / 브라우저 등

### PR DESCRIPTION
**Pull request by ssm0318**
## Self Check List

- [x] !!! PR이 머지되는 base branch을 꼭 확인해주세요 !!!
- [x] base branch로부터 rebase를 했나요?

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (formatting, renaming)
- [ ] Merge Develop to Main
- [ ] Other (please describe):

### CODE REVIEW

File: adoorback/qna/views.py [hunk 1]
```diff
@@ -94,7 +94,8 @@ def get_exception_handler(self):
 
     def get_queryset(self):
         current_user = self.request.user
-        return Response.objects.get(id=self.kwargs.get('pk')).response_comments.exclude(author_id__in=current_user.user_report_blocked_ids)
+        return Response.objects.get(id=self.kwargs.get('pk')).response_comments.exclude(
+            author_id__in=current_user.user_report_blocked_ids).order_by('-created_at')
```
Comment (koyrkr) [inline adoorback/qna/views.py:98]: response까지 확인해주시고 수정해주셔서 감사합니다! 🙇‍♀️

**Review by koyrkr — APPROVED** [review]
👍

File: adoorback/note/views.py [hunk 2]
```diff
@@ -34,13 +34,15 @@ def perform_create(self, serializer):
 class NoteComments(generics.ListAPIView):
     serializer_class = cs.CommentFriendSerializer
     permission_classes = [IsAuthenticated, IsNotBlocked]
+    # ordering = ['-created_at']
```
Comment (njs03332) [inline adoorback/note/views.py:37]: 요건 주석처리된 것 같아서 삭제해도 될것 같아용

**Review by njs03332 — APPROVED** [review]
굿굿임니당

**Pull request merged**